web-wallet/src/lib/components/OperationResult/OperationResult.svelte

```diff
@@ -1,89 +1,80 @@
-<svelte:options immutable={true}/>
+<svelte:options immutable={true} />
 
 <script>
-	import {
-		AnchorButton,
-		ErrorDetails,
-		Icon,
-		Throbber
-	} from "$lib/dusk/components";
-	import { mdiCheckDecagramOutline, mdiCloseThick } from "@mdi/js";
-	import { makeClassName } from "$lib/dusk/string";
+  import {
+    AnchorButton,
+    ErrorDetails,
+    Icon,
+    Throbber,
```

Comment:
Again this extra comma, do not know how it is added as it does not happen constantly as you can see 2 lines below.